ban/http/api.py

```diff
@@ -657,7 +657,7 @@ def get_collection(self):
 
 @app.route('/openapi', methods=['GET'])
```

Comment:
I guess you planned to add the jsonify decorator here? ;)

Which shows that a unit test is needed ;)